### ISSUE TITLE
FQ exception classname is declared though alias used

### ISSUE THREAD

**Issue by CZechBoY** [kalessil/phpinspectionsea#453]
Hi,
I have this code
```php
namespace Abc;

use Def\MyException;

class AbcClass
{
    public function myMethod()
    {
        throw new MyException(); // Throws a non-annotated/unhandled exception: 'Def\MyException'.
    }
}
```
This is ok.
But if I click to auto-fix `Declare exception via @throws`, then it add phpdoc containing fq class name `\ Def\MyException` instead of just `MyException` as used in throw statement.

It is not bug, but it is annoying to always removed that namespace from phpdoc :-).

**Comment by funivan** [comment]
Thanks for the report. Reproduced. 
I think we can fix this.

**Comment by funivan** [comment]
@kalessil when we use aliases maybe we should alias name for the `@throws` tag  
```php
namespace Abc {
    use Def\MyException as MyCustomEx;
    class AbcClass {

     /**
       * @throws MyCustomEx   << Should be like this 
       */
      public function myMethod() {
        throw new MyCustomEx("Msf"); 
      }
    }
  }
```

**Comment by CZechBoY** [comment]
Any progress here?

**Comment by kalessil** [comment]
I'll add an option for QF style, scheduling for 2.3.12 but still can shift this - false-positives report rate is high.

**Comment by kalessil** [comment]
The inspection has been dropped this year, hence the ticket is outdated.
